loadtest/user/userentity/actions.go

```diff
@@ -426,6 +426,15 @@ func (ue *UserEntity) GetEmojiList(page, perPage int) error {
 	return ue.store.SetEmojis(emojis)
 }
 
+func (ue *UserEntity) SaveReaction(reaction *model.Reaction) error {
+	r, resp := ue.client.SaveReaction(reaction)
+	if resp.Error != nil {
+		return resp.Error
+	}
+
+	return ue.store.SetReaction(r)
```

Comment:
No need to touch the store here. The store is going to get initialized on `GetReactions` and updated via websocket.